src/components/Homepage/Banner.js

```diff
@@ -0,0 +1,9 @@
+function Banner() {
+  return (
+    <section className="flex flex-col justify-center items-center bg-[#5cb85c] text-white h-32">
```

Comment:
Have the color [#5cb85c] be a global variable since it is likely to be used again.

Reply:
I will add that color on tailwindcss as "main-green"